composables/masto/routes.ts

```diff
@@ -33,11 +33,11 @@ export function getReportRoute(id: string | number) {
   return `https://${currentUser.value?.server}/admin/reports/${encodeURIComponent(id)}`
 }
 
-export function getStatusRoute(status: mastodon.v1.Status) {
+export function getStatusRoute(status: mastodon.v1.Status, singleInstance?: boolean) {
```

Comment:
rather than taking this as a parameter, I think it's better to just determine this at the top of the file. There are a few other places where `getStatusRoute` is used in Elk, and this way we would be able to update the behavior of those as well (without going and manually adding the extra param everywhere it is used).

Elk already has a way to determine singleInstance-ness:
```
const { singleInstanceServer } = useSignIn()
```
which is just a boolean

Reply:
Word.  As long as it's not an anti-pattern in elk/vue I am good with this.

Reply:
I am actively approaching this with kid gloves so as to not rock the boat to hard with changes that will propagate everywhere, but I am all good with it if everyone else is on board ⛵ 🪨

Reply:
ok I've looked more thoroughly at the other places that `getStatusRoute` is used, and I'm more sure that it's safe to proaogate the change everywhere.

Rather than doing `const { singleInstanceServer } = useSignIn()` at the top of the file, it can just be the first line in `getStatusRoute`. That will be more idiomatic now that I think of it.